From: Debian Cinnamon Team <pkg-cinnamon-team@lists.alioth.debian.org>
Date: Mon, 28 Mar 2016 11:56:35 +0200
Subject: libsystemd

---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 7afbfbf..4b9b1de 100644
--- a/configure.ac
+++ b/configure.ac
@@ -373,7 +373,7 @@ dnl ====================================================================
 dnl Check for logind
 dnl ====================================================================
 
-PKG_CHECK_MODULES(LOGIND, [libsystemd-login], [have_logind=yes], [have_logind=no])
+PKG_CHECK_MODULES(LOGIND, [libsystemd], [have_logind=yes], [have_logind=no])
 
 if test x$have_logind = xyes; then
     AC_DEFINE(HAVE_LOGIND, 1, [Define if logind is supported])
